app/flow.py

```diff
@@ -1,12 +1,16 @@
 import zmq
 from .json_util import to_json
+from .env import load_env
+
+
+env, cfg = load_env()
```

Comment:
@stefanv can you think of any issues w/ loading the config at import time like this?

Reply:
Should be fine, we no longer rely on magical things happening in the background.